data/ooniprobe.conf.sample

```diff
@@ -4,8 +4,19 @@
 
 basic:
     # Where OONIProbe should be writing its log file
-    logfile: /var/log/ooniprobe.log
-    loglevel: WARNING
+    # logfile: {logfile}
+    # loglevel: WARNING
+    # Will rotate logs daily when "daily" or based on rotate_length
+    #  when "length" is set. null to disable rotation.
+    # rotate: daily
+    # rotate_length: 1M
+    # rotate: length
+    # Sets an upper bound on the number of rotated files, only works when
+    #  "length" is the rotation mode
+    # max_rotated_files: None
```

Comment:
It's `null` in YAML, not `None`.

I'm unsure if oo.conf is parsed with yaml parser or not, but it looks like YAML.